src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/ExecuteTaskReconfigurationService.java

```diff
@@ -144,6 +145,14 @@ private List<String> reconfigureTasks(List<String> taskIds, List<TaskResource> s
         return failedTaskIds;
     }
 
+    private void resetIndexed(TaskResource taskResource) {
+        if (!taskResource.getIndexed()
+            && (taskResource.getState() == CFTTaskState.ASSIGNED
+                || taskResource.getState() == CFTTaskState.UNASSIGNED)) {
+            taskResource.setIndexed(true);
```

Comment:
Maybe put an info log in here so you can see in envs that it was false and is now being made true

Reply:
Thank you for the suggestion Ravi... I added the info logs.

Reply:
👍